api/swagger.yaml

```diff
@@ -8861,6 +8861,13 @@ paths:
                   nodes in order to reach the containers running on this node. Using this parameter it is possible to
                   separate the container data traffic from the management traffic of the cluster.
                 type: "string"
+              DataPathPort:
```

Comment:
See my other comment (https://github.com/moby/moby/pull/38102/files#r233649721) looks like the new field in `ClusterInfo` was not yet added to the swagger; can you update the swagger? Here's a diff that you can apply;

```patch
diff --git a/api/swagger.yaml b/api/swagger.yaml
index 5b721ee82c..f12cde5439 100644
--- a/api/swagger.yaml
+++ b/api/swagger.yaml
@@ -2465,6 +2465,16 @@ definitions:
         description: "Whether there is currently a root CA rotation in progress for the swarm"
         type: "boolean"
         example: false
+      DataPathPort:
+        description: |
+          DataPathPort specifies the data path port number for data traffic.
+          Acceptable port range is 1024 to 49151.
+
+          If no port is set or is set to 0, the default port (4789) is used.
+        type: "integer"
+        format: "uint32"
+        default: 4789
+        example: 4789
 
   JoinTokens:
     description: |
```